test/Interop/Cxx/deserialize-debug.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir -g | %FileCheck %s
+
+// Validate that we don't crash when trying to deserialize C++ type debug info.
+
+import IntBox
+
+func blackHole<T>(_ x : T) -> T { x }
+
+public func create(_ x : Int32) -> IntBox {
+  return IntBox(x: x)
+}
+
+// CHECK-LABEL: define swiftcc void @"$s4main4testyyF"
+// CHECK: [[P:%.*]] = alloca %Ts5Int32V
+// CHECK: [[E:%.*]] = getelementptr inbounds %Ts5Int32V, %Ts5Int32V* [[P]]
+// CHECK: [[V:%.*]] = load i32, i32* [[E]]
+// CHECK: call swiftcc i32 @"$s4main6createySo6IntBoxVs5Int32VF"(i32 [[V]])
+// CHECK: ret void
+public func test() {
+  let f = create(Int32.random(in: 0..<6))
```

Comment:
Is `Int32.random()` critical here? Can we replace it by a constant, or by a function in this file that returns a constant?